backend/backends/pytorch_backend.py

```
@@ -331,40 +342,46 @@ async def transcribe(
         """
         await self.load_model_async(model_size)
 
+        progress_model_name = f"whisper-{self.model_size}"
+
         def _transcribe_sync():
             """Run synchronous transcription in thread pool."""
             # Load audio
             audio, sr = load_audio(audio_path, sample_rate=16000)
```

Ruff (0.15.10)

[warning] Unpacked variable `sr` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)